db/structure.sql

```diff
@@ -203,13 +203,46 @@ ALTER SEQUENCE public.attached_files_id_seq OWNED BY public.attached_files.id;
 
 
 --
--- Name: collections; Type: TABLE; Schema: public; Owner: -
+-- Name: collection_versions; Type: TABLE; Schema: public; Owner: -
 --
 
-CREATE TABLE public.collections (
+CREATE TABLE public.collection_versions (
     id bigint NOT NULL,
+    version integer DEFAULT 1 NOT NULL,
+    state character varying NOT NULL,
     name character varying NOT NULL,
     description character varying,
+    collection_id bigint,
+    created_at timestamp(6) without time zone NOT NULL,
+    updated_at timestamp(6) without time zone NOT NULL
+);
+
+
+--
+-- Name: collection_versions_id_seq; Type: SEQUENCE; Schema: public; Owner: -
+--
+
+CREATE SEQUENCE public.collection_versions_id_seq
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+
+--
+-- Name: collection_versions_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: -
+--
+
+ALTER SEQUENCE public.collection_versions_id_seq OWNED BY public.collection_versions.id;
+
+
+--
+-- Name: collections; Type: TABLE; Schema: public; Owner: -
+--
+
+CREATE TABLE public.collections (
+    id bigint NOT NULL,
     release_option character varying,
```

Comment:
Is there a reason why properties like `release_option`, `release_duration`, `access`, `required_license`, are in the collection model verses in the collection_version model?

Reply:
Yes, because those are now just settings and they don't go in the repository. They are not versioned.